### PR TITLE
[FUSION-15] removed tvOS example project and documentation

### PR DESCRIPTION
**Pull request by akamimoor**
## JIRA ticket
[FUSION-15](https://janrain.atlassian.net/browse/FUSION-15)

## Changes

- Removed the tvOS sample app
- Removed the tvOS documentation

Confluence doc for development instructions: [Here](https://janrain.atlassian.net/l/cp/wqc2506v)

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -57,29 +53,12 @@ let package = Package(
...
-            path: "Source/AppAuthTV",
-            publicHeadersPath: ""
-        ),
-        .testTarget(
-            name: "AppAuthCoreTests",
-            dependencies: ["AppAuthCore"],
-            path: "UnitTests",
-            exclude: ["OIDSwiftTests.swift", "AppAuthTV"]
-        ),
+        )
```
Comment (kreese-akamai) [inline Package.swift:56]:
```suggestion
        ),
```

**Review by kreese-akamai — DISMISSED** [review]
Also, needs some changes in `.github/workflows/tests.yml`
see: https://github.com/janrain/AppAuth-iOS/pull/10/files

**Review by schaeferCode — APPROVED** [review]
Seems pretty good. I'm good with the changes assuming @kreese-akamai 's comments are addressed.

**Comment by akamimoor** [comment]
Looks good to me!

File: Package.swift [hunk 2]
```diff
@@ -58,28 +54,11 @@ let package = Package(
...
-            name: "AppAuthTV",
-            dependencies: ["AppAuthCore"],
-            path: "Source/AppAuthTV",
-            publicHeadersPath: ""
-        ),
-        .testTarget(
-            name: "AppAuthCoreTests",
-            dependencies: ["AppAuthCore"],
-            path: "UnitTests",
-            exclude: ["OIDSwiftTests.swift", "AppAuthTV"]
```
Comment (AnyaDKing) [inline Package.swift:71]: Should this be removed entirely, or just the `AppAuthTv` portion?

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -12,16 +12,13 @@ on:
 jobs:
 
   xcode-project-test:
-    runs-on: macOS-latest
+    runs-on: macOS-11
```
Comment (AnyaDKing) [inline .github/workflows/tests.yml:15]: Was this for testing? Or should this be changed?

Comment (kreese-akamai) [reply .github/workflows/tests.yml:15]: GitHub released testing image `macOS-12` recently _(this past week)_. Previously `macOS-latest` was spinning up an instance of `macOS-11`, but now `macOS-latest` runs `macOS-12`. For one reason or another, the AppAuth test suite is not compatible without additional changes. This is apparent in [master branch commit history](https://github.com/janrain/AppAuth-iOS/commits/master), where no code changes occur other than clean merges. You can also see which unit tests pass and fail in this [testing branch](https://github.com/janrain/AppAuth-iOS/pull/10) clicking on the red **x** by each commit sha.

Comment (kreese-akamai) [reply .github/workflows/tests.yml:15]: We are also several commits behind the upstream [openid/AppAuth-iOS)](https://github.com/openid/AppAuth-iOS) repo off which we are forked. They will likely need to update their GitHub workflows and/or test suites to support `macOS-12`. That may be a good time for us to merge in their master branch into our master branch.

File: Package.swift (hunk 2 above)
Comment (kreese-akamai) [reply Package.swift:71]: Good catch! This should be restored:
```swift
.testTarget(
    name: "AppAuthCoreTests",
    dependencies: ["AppAuthCore"],
    path: "UnitTests",
    exclude: ["OIDSwiftTests.swift"]
),
```
Verified with [Test Commit](https://github.com/janrain/AppAuth-iOS/pull/10/commits/ef0d11fae30bb4e4413b61223e7b63116bbd2cb3)

**Review by kreese-akamai — CHANGES_REQUESTED** [review]
proposed changes in [Test Pull Request](https://github.com/janrain/AppAuth-iOS/pull/10/file)

**Review by jbush-janrain — APPROVED** [review]

**Pull request merged**